app/controllers/feeds_controller.rb

```diff
@@ -85,6 +85,15 @@ def feed_params
       :display_episodes_count,
       :display_full_episodes_count,
       :episode_offset_seconds,
+      :audio_type,
+      :audio_bitrate,
+      :audio_bitdepth,
+      :audio_channel,
+      :audio_sample,
+      :billboard,
+      :house,
+      :paid,
+      :sonic_id,
       feed_tokens_attributes: %i[id label token _destroy],
```

Comment:
Unrelated to this PR request: can you sort the Feeds tabs by something?  Maybe just `created_at: :asc`, so they show in the order you created them?  Just a bit jarring to compare with Publish when they're in a different order.